homeassistant/components/insteon_plm.py

```diff
@@ -0,0 +1,20 @@
+"""
+Support for INSTEON PowerLinc Modem.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/insteon_plm/
+"""
+import asyncio
+import logging
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@asyncio.coroutine
+def async_setup(hass, config):
+    """Setup the insteon_plm component."""
+    _LOGGER.warning('The insteon_plm comonent has been replaced by '
```

Comment:
Make it a persistent notification for more visibility.

Reply:
Good idea. I added that to both insteon_plm and insteon_local.